src/webui/www/private/confirmdeletion.html

```diff
@@ -45,9 +45,9 @@
     <br />
 
     <p>&nbsp;&nbsp;QBT_TR(Are you sure you want to delete the selected torrents from the transfer list?)QBT_TR[CONTEXT=HttpServer]</p>
```

Comment:
Here is another set of ideas:
```suggestion
    <p>&nbsp;&nbsp;QBT_TR(Are you sure you want to remove the selected torrents from the transfer list?)QBT_TR[CONTEXT=HttpServer]</p>
```
Use `remove` to avoid users mix up with deleting files.